lib/presentation/widget/cooking_time.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+import '../../core/util/asset_constants.dart';
+import '../../core/util/dimensions.dart';
+import '../../core/util/palette.dart';
+import '../../core/util/text_constants.dart';
+
+class CookingTime extends StatelessWidget {
+  final int time;
+
+  late final int hours;
+  late final int minutes;
+
+  CookingTime({Key? key, required this.time}) : super(key: key) {
+    hours = time ~/ Dimensions.minutesInHour;
+    minutes = time % Dimensions.minutesInHour;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Tooltip(
+      message:
+          'Ready in ${hours > 0 ? ('$hours ${hours == 1 ? 'hour' : 'hours'} and') : ''} $minutes ${minutes == 1 ? 'minute' : 'minutes'}.',
```

Comment:
Also if you choose the TimeOfDay approach this could be an extension function for this class

Reply:
Done.